spec/facades/photo_facade_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Photo Facade' do
+  describe 'Photo', :vcr do
+    it 'makes an API call using the Unsplash gem' do
+      response = PhotoFacade.photos("cat")
+      response_json = response.to_json
+      parse = JSON.parse(response_json, symbolize_names: true)
+
+      expect(parse).to be_a Array
+      parse.each do |photo|
+        expect(photo).to be_a Hash
+      end
+    end
+    it 'returns a single Unsplash photo' do
+      single_response = PhotoFacade.single_photo
+      single_photo = JSON.parse(single_response, symbolize_names: true)
+      
+      expect(single_photo).to be_a Hash
+      expect(single_photo).to have_key (:attributes)
+      expect(single_photo[:attributes]).to be_a Hash
+      expect(single_photo[:attributes]).to have_key (:table)
+      expect(single_photo[:attributes][:table]).to be_a Hash
+      expect(single_photo[:attributes][:table]).to have_key (:id)
+      expect(single_photo[:attributes][:table][:id]).to be_a String
+      expect(single_photo[:attributes][:table]).to have_key (:urls)
+      expect(single_photo[:attributes][:table][:urls]).to be_a Hash
+      expect(single_photo[:attributes][:table][:urls]).to have_key (:raw)
+      expect(single_photo[:attributes][:table][:urls][:raw]).to be_a String
+      expect(single_photo[:attributes][:table][:urls][:raw]).to include "https://images.unsplash.com/photo"
+      
```

Comment:
consider adding a commented out note that indicates what the point of digging through :users is (bc we want to give photo credit)

Reply:
Yes, I love the idea of giving photo credit; this company is too small to get entangled with any sort of litigations. And having the developer empathy to easily identify what we're doing with :users would be a great add.